content/en/docs/distributions/gke/deploy/upgrade.md

```diff
@@ -1,31 +1,103 @@
 +++
 title = "Upgrade Kubeflow"
 description = "Upgrading your Kubeflow installation on Google Cloud"
-weight = 5
+weight = 6
 +++
 
 ## Before you start
 
 To better understand upgrade process, you should read the following sections first:
 
-* [Understanding the deployment process for management cluster](../management-setup#understanding-the-deployment-process)
-* [Understanding the deployment process for Kubeflow cluster](../deploy-cli#understanding-the-deployment-process)
+* [Understanding the deployment process for management cluster](/docs/gke/deploy/management-setup#understanding-the-deployment-process)
+* [Understanding the deployment process for Kubeflow cluster](/docs/gke/deploy/deploy-cli#understanding-the-deployment-process)
 
 This guide assumes the following settings:
 
 * The `${MGMT_DIR}` and `${MGMT_NAME}` environment variables
-  are the same as in [Management cluster setup](../management-setup#environment-variables).
+  are the same as in [Management cluster setup](/docs/gke/deploy/management-setup#configure-environment-variables).
 * The `${KF_DIR}`, `${KF_NAME}`, `${CLIENT_ID}` and `${CLIENT_SECRET}` environment variables
-  are the same as in [Deploy using kubectl and kpt](../deploy-cli#environment-variables).
+  are the same as in [Deploy using kubectl and kpt](/docs/gke/deploy/deploy-cli#environment-variables).
 
 ## General upgrade instructions
 
-Both management cluster and Kubeflow cluster follow the same `instance` and `upstream` folder convention. To upgrade, you'll typically need to update packages in `upstream` to the new version and repeat the `make apply-<subcommand>` commands in their respective deployment process.
+Starting from Kubeflow v1.3, we have reworked on the structure of `kubeflow/gcp-blueprints` repository. All resources are located in `gcp-blueprints/management` directory.
+
+Before Kubeflow v1.3, both management cluster and Kubeflow cluster follow the same `instance` and `upstream` folder convention. To upgrade, you'll typically need to update packages in `upstream` to the new version and repeat the `make apply-<subcommand>` commands in their respective deployment process.
 
 However, specific upgrades might need manual actions below.
 
 ## Upgrading management cluster
 
+### General instruction for upgrading management cluster
+
+It is strongly recommended to use source control to keep a copy of your working repository for recording changes at each step.
+
+Due to the refactoring of `kubeflow/manifests` repository, the way we depend on `kubeflow/gcp-blueprints` has changed drastically. This section suits for upgrading from Kubeflow 1.3 to higher.
```

Comment:
Shall we mention upgrade from 1.2 to 1.3 is not supported?

Reply:
Added this notion in earlier part of this section.